libs/types/src/tokens.rs

```diff
@@ -210,6 +199,61 @@ pub struct CustomMetadata {
 
 	/// Whether an asset can be used as a currency to fund Centrifuge Pools.
 	pub pool_currency: bool,
+
+	/// The ways, if any, this token is cross-chain transferable
+	pub transferability: Option<CrossChainTransferability>,
+}
+
+impl Default for CustomMetadata {
+	fn default() -> Self {
+		Self {
+			xcm: Default::default(),
+			mintable: false,
+			permissioned: false,
+			pool_currency: false,
+			transferability: Some(CrossChainTransferability::Xcm),
+		}
+	}
+}
+
+/// The Cross Chain Transferability property of an asset describes the way(s),
+/// if any, that said asset is cross-chain transferable. It may currently be
+/// transferable through Xcm, Centrifuge Connectors, or All .
+///
+/// NOTE: Once set to `All`, the asset is automatically transferable through any
+/// eventual new option added at a later stage. A migration might be required if
+/// that's undesirable for any registered asset.
+#[derive(
+	Clone,
+	Copy,
+	Default,
+	PartialOrd,
+	Ord,
+	PartialEq,
+	Eq,
+	Debug,
+	Encode,
+	Decode,
+	TypeInfo,
+	MaxEncodedLen,
+)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+pub enum CrossChainTransferability {
+	/// The asset is only transferable through XCM
+	#[default]
```

Comment:
Is this default used only for tests/benchmarks, or does it make sense to have `Xcm` as the expected/default behavior in production?

I say this because it's easier to initialize this value without realizing it by default with a non-specified value in production code.

Reply:
Both, actually. By default, a token registered in the registry is XCM transferrable unless set otherwise. And for tests it's also very useful to have this default precisely because that's the default property.

Reply:
> I am open to that, and thanks for the suggestion, but do you have an example of where it's dangerous to have the default being Some(XCM)? Because having the default set to None is neither helpful for testing purposes neither useful in practise. Let me know 👍

Lets say we add registry default entries for all currencies and then test networks will be different. And it feels like too much allowance for a default to be transferable over xcm. But maybe just me not trusting xcm setups. 

Again, if it is only a default for testing, I would hide it behind std feature, so it does not hit the wasm.

Reply:
But not blocking of course. Just the typical reviewing eye ^^

Reply:
No, this is a good point. I added the `None` variant back to `CrossChainTransferability` and have that being the default.